state/protocol/badger/state.go

```diff
@@ -129,15 +131,22 @@ func Bootstrap(
 		return nil, fmt.Errorf("could not get sealing segment: %w", err)
 	}
 
+	_, rootSeal, err := root.SealedResult()
+	if err != nil {
+		return nil, fmt.Errorf("could not get sealed result for sealing segment: %w", err)
+	}
+
 	err = operation.RetryOnConflictTx(db, transaction.Update, func(tx *transaction.Tx) error {
 		// sealing segment is in ascending height order, so the tail is the
 		// oldest ancestor and head is the newest child in the segment
 		// TAIL <- ... <- HEAD
-		highest := segment.Highest() // reference block of the snapshot
-		lowest := segment.Sealed()   // last sealed block
+		lastFinalized := segment.Highest() // the highest block in sealing segment is the last finalized block
```

Comment:
can we also add this comment to the `seling_segment.go`

Reply:
We could also add an alias for `Highest` called `Finalized` to SealingSegment (?)